src/containers/MPT/styles.scss

```diff
@@ -0,0 +1,15 @@
+@import '../shared/css/variables';
+
+.mpt-page {
```

Comment:
Not necessarily needed in this PR but we should create a template for token pages so that there'd be not a lot of copy & paste needed